### PR TITLE
Mbjson split

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
  * [x] Bug fix
  * [ ] Feature addition
  * [ ] Refactoring
  * [ ] Minor / simple change (like a typo)
  * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

### CODE REVIEW

File: picard/mbjson.py
Comment (phw) [inline picard/mbjson.py:223]: What do you think about using a namedtuple for this?

File: picard/mbjson.py
Comment (phw) [inline picard/mbjson.py:232]: `funcs` does not change, but `_relations_to_metadata`  is called quite frequently. I think we could rather have a dict outside the function in the module. I even think it would be more readable to have an explicit, hard coded mapping of target-type name to function.

**Review by phw — CHANGES_REQUESTED** [review]
Definitive an improvement. I'd suggest to merge this after we merged #2213, though, to keep the diff more focused on the actual change.

**Comment by zas** [comment]
I'll redo that after the merge

**Pull request closed without merging**
